packages/babel-preset-expo/src/index.ts

```diff
@@ -163,6 +165,12 @@ function babelPresetExpo(api: ConfigAPI, options: BabelPresetExpoOptions = {}):
 
   const platformOptions = getOptions(options, platform);
 
+  // If the input is a script, we're unable to add any dependencies. Since the @babel/runtime transformer
+  // adds extra dependencies (requires/imports) we need to disable it
+  if (type === 'script' && platformOptions.enableBabelRuntime !== false) {
```

Comment:
```suggestion
  if (type === 'script') {
```

nit: same reason as above